docs/knowledgebase/integrate/subkey.md

```diff
@@ -140,7 +136,7 @@ The `inspect` command recalculates a key pair's public key and public address gi
 This shows that it is sufficient to back up the seed alone.
 
 ```bash
-$ subkey inspect 0x554b6fc625fbea8f56eb56262d92ccb083fd6eaaf5ee9a966eaab4db2062f4d0
+$ subkey inspect-key --uri 0x554b6fc625fbea8f56eb56262d92ccb083fd6eaaf5ee9a966eaab4db2062f4d0
```

Comment:
```
> subkey inspect-key --uri 0x554b6fc625fbea8f56eb56262d92ccb083fd6eaaf5ee9a966eaab4db2062f4d0
error: Found argument '--uri' which wasn't expected, or isn't valid in this context

USAGE:
    subkey inspect-key [FLAGS] [OPTIONS] [uri]

For more information try --help
```

Reply:
Hmmmm not sure what's going on here. This is what I experience:
```shell
~: subkey --version
subkey 2.0.0-rc6
~: subkey inspect-key --uri 0x554b6fc625fbea8f56eb56262d92ccb083fd6eaaf5ee9a966eaab4db2062f4d0
Secret Key URI `0x554b6fc625fbea8f56eb56262d92ccb083fd6eaaf5ee9a966eaab4db2062f4d0` is account:
  Secret seed:      0x554b6fc625fbea8f56eb56262d92ccb083fd6eaaf5ee9a966eaab4db2062f4d0
  Public key (hex): 0x143fa4ecea108937a2324d36ee4cbce3c6f3a08b0499b276cd7adb7a7631a559
  Account ID:       0x143fa4ecea108937a2324d36ee4cbce3c6f3a08b0499b276cd7adb7a7631a559
  SS58 Address:     5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd
```